test/unit_test/workflows/mock_data/text_generation_dataset_random.json

```diff
@@ -0,0 +1,62 @@
+{
+    "input_model":{
+        "type": "PyTorchModel",
+        "config": {
+            "hf_config": {
+                "model_name": "gpt2",
+                "task": "text-generation",
+                "dataset": {
+                    "data_name": "ptb_text_only",
```

Comment:
There is the disk limitation in our pipeline. As this case will download new data which may bring the risk to trigger `out of space` error.

How about use bert model + glue dataset for test?

Reply:
There is another workflow which uses the same dataset https://github.com/microsoft/Olive/blob/main/test/unit_test/workflows/mock_data/text_generation_dataset.json. This is the same dataset with just some new args. 

So I think this might be okay? Also, I think PTB dataset is not too big.